### ISSUE TITLE
Fix UIPasteboard warnings

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#17181]
```
⚠️  /Users/vagrant/git/Client/Extensions/UIPasteboard+Extension.swift:15:13: 'kUTTypeURL' was deprecated in iOS 15.0: Use UTTypeURL or UTType.url (swift) instead.
            kUTTypeURL as String: url,
            ^
⚠️  /Users/vagrant/git/Client/Extensions/UIPasteboard+Extension.swift:21:25: 'kUTTypeGIF' was deprecated in iOS 15.0: Use UTTypeGIF or UTType.gif (swift) instead.
        return (isGIF ? kUTTypeGIF : kUTTypePNG) as String
                        ^
⚠️  /Users/vagrant/git/Client/Extensions/UIPasteboard+Extension.swift:21:38: 'kUTTypePNG' was deprecated in iOS 15.0: Use UTTypePNG or UTType.png (swift) instead.
        return (isGIF ? kUTTypeGIF : kUTTypePNG) as String
        
⚠️  /Users/vagrant/git/Tests/ClientTests/Extensions/UIPasteboardExtensionsTests.swift:28:66: 'kUTTypePNG' was deprecated in iOS 15.0: Use UTTypePNG or UTType.png (swift) instead.
        verifyPasteboard(expectedURL: url, expectedImageTypeKey: kUTTypePNG)
                                     ^
⚠️  /Users/vagrant/git/Tests/ClientTests/Extensions/UIPasteboardExtensionsTests.swift:36:66: 'kUTTypeGIF' was deprecated in iOS 15.0: Use UTTypeGIF or UTType.gif (swift) instead.
        verifyPasteboard(expectedURL: url, expectedImageTypeKey: kUTTypeGIF)
                                                                 ^
⚠️  /Users/vagrant/git/Tests/ClientTests/Extensions/UIPasteboardExtensionsTests.swift:42:44: 'kUTTypeURL' was deprecated in iOS 15.0: Use UTTypeURL or UTType.url (swift) instead.
        XCTAssertEqual(pasteboard.items[0][kUTTypeURL as String] as? URL, expectedURL)
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-7701)

**Comment by data-sync-user** [comment]
➤ Winnie Teichmann commented:

For QA: Please check that the pasteboard still works correctly.

**Comment by data-sync-user** [comment]
➤ Diana Andreea Barladeanu commented:

Verified as fixed on v121(36341), with iPhone SE (16.6.1).
